packages/contracts-bedrock/contracts/test/FeeVault.t.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Bridge_Initializer } from "./CommonTest.t.sol";
+
+import { L1FeeVault } from "../L2/L1FeeVault.sol";
+import { BaseFeeVault } from "../L2/BaseFeeVault.sol";
+import { StandardBridge } from "../universal/StandardBridge.sol";
+import { Predeploys } from "../libraries/Predeploys.sol";
+
+// Test the implementations of the FeeVault
+contract BFeeVault_Test is Bridge_Initializer {
+    BaseFeeVault baseFeeVault = BaseFeeVault(payable(Predeploys.BASE_FEE_VAULT));
+    L1FeeVault l1FeeVault = L1FeeVault(payable(Predeploys.L1_FEE_VAULT));
+
+    address constant recipient = address(0x10000);
+
+    function setUp() public override {
+        super.setUp();
+        vm.etch(Predeploys.BASE_FEE_VAULT , address(new BaseFeeVault(recipient)).code);
+        vm.etch(Predeploys.L1_FEE_VAULT , address(new L1FeeVault(recipient)).code);
```

Comment:
is this spacing before the `,` intentional? I'm surprised the formatter doesn't remove it, unless it's not actually operating on the test files despite #3663?

Reply:
That is a typo, good catch, will fix